### ISSUE TITLE
circleColor - Hit an exception when attempting to set its value

### ISSUE THREAD

**Issue by baldscone** [DavydLiu/DLRadioButton#11]
Hi there,
I am just updating my project to xCode7 and iOS9 and encountered this error for every instance of DLRadioButton in my Storyboard? It also occurs for circleRadius. Setting them manually to the default and restarting xCode didn't work. Any ideas or ideas for a fix? BTW I am using Swift 2.0 / iOS9 / xCode7

```
file:///blah/blah/Base.lproj/Main.storyboard: warning: IB Designables: Ignoring user defined runtime attribute for key path "circleColor" on instance of "DLRadioButton". Hit an exception when attempting to set its value: [<DLRadioButton 0x7fd4f245b220> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key circleColor.
```

Seems like because they are set to default it triggers an error. When I set them all manually to a default value in IB I get another error as below:

```
file:///blah/blah/Base.lproj/Main.storyboard: error: IB Designables: Failed to update auto layout status: The agent crashed

file:///blah/blah/Base.lproj/Main.storyboard: error: IB Designables: Failed to render instance of DLRadioButton: The agent crashed
```

I have tried clearing my Derived Data folder but warnings persist.

**Comment by DavydLiu** [comment]
Thanks for open this issue. Unfortunately, both `circleRadius` and `circleColor` are deprecated when DLRadioButton updated to 1.3.0. The best thing you can do is update to the latest 1.3.5 version.

**Comment by baldscone** [comment]
Thanks for the prompt reply - I am using DLRadioButton 1.3.5 via Cocoapods. Do you mean that laying out the buttons again in my Storyboard might fix the issue? Is there any fix or should I just ignore warning?

**Comment by DavydLiu** [comment]
Welcome, `circleRadius` and `circleColor` were removed when I updated DLRadioButton to 1.3.0. You should not see any reference to those two properties. I'd suggest you clean you project and try to build again.

**Comment by baldscone** [comment]
OK understood, thanks for your reply. I have tried cleaning the project and building again but to no avail. I tried with one instance and the only fix seems to be deleting all the instances and relaying out the storyboard. I have about 30 radio buttons so its a big task. Cleaning build folder also doesn't work - not sure where it is caching the references to circleRadius and circleColor? Any suggestions welcome.
